### DISCUSSION TITLE
Tasmota Sonnoff Basic - Toggle

### DISCUSSION THREAD

**Discussion by Fokker-1** [arendst/Tasmota#21928] [Q&A]
Hallo,

ich betreibe meinen eHz mit einem Hichi Wifi IR-Leser zum Auslesen der Zählerdaten. Die Verbrauchs-Anzeigen erfolgen im Web-Browser. Das funktioniert alles prima.

Bei der Webansicht:

![Tasmota_Screen mit Spg_verdeckt](https://github.com/user-attachments/assets/5b34f60e-4ec5-43ce-b5ad-79f4c3cd133c)

benötige ich die Anzeigen "Toggle" und dazu dann auch "ON / OFF" nicht und möchte diese gerne von der Oberfläche entfernen.

Kann mir da jemand hier bitte Hinweise geben?
Meine eigenen Kenntnisse sind doch sehr begrenzt, so dass werkeln im Code und Neuflashen wohl eher ausscheiden.

Schon mal danke im Voraus und Grüße
Werner

**Comment by stefanbode** [comment]
Schau mal in der configuration ob dort ein RELAY1 definiert ist. Wenn ja merken was da steht und den entsprechenden Port auf NONE stellen. Reboot und testen ob noch alles da ist. Feedback erwünscht.

**Reply by Fokker-1** [reply]
Guten Morgen,

danke für den Hinweis.

Ich habe in allen Bereichen vo "configuration" nachgesehen: ein RELAY1 habe ich nicht gefunden.
Den Begriff RELAY sah ich auch nirgendwo.

Gruß
Werner

**Reply by stefanbode** [reply]
Ok dann ist es vielleicht ein Template. Ich schaue gleich mal und sende Screenshots wo du das dann ändern kannst. Als Antwort bitte auch Screenshots senden. Danke

**Reply by Fokker-1** [reply]
Danke schon mal!

**Reply by Jason2866** [reply]
Das Device sollte nicht als "Sonoff Basic" definiert werden. Da ist selbstverständlich ein Relais dabei.

**Reply by Fokker-1** [reply]
Danke für die Info.

Wie schon geschrieben, kenne ich mich hier nicht wirklich aus und habe das (noch) nicht verstanden:

**_"Das Device sollte nicht als "Sonoff Basic" definiert werden."_**

Was genau ist damit gemeint?

Ich habe diesen Hichi-IR-Leser gekauft, der kam fertig konfiguriert, da habe ich also nichts selbst geflasht:

![Hichi IR_Web_1000](https://github.com/user-attachments/assets/bc597491-21f2-46f7-9fd8-c684acea2765)

Lediglich das Script für den spezifischen eHZ habe ich aus GitHub bei "consoles / edit script" noch reinkopiert.
Das läuft ja auch alles. Auch die Bezeichnung "Sonoff Basic" (siehe ganz oben) war so vorgegeben.

Du schreibst weiter:

**_"Da ist selbstverständlich ein Relais dabei."._**

Ich habe jetzt noch einaml unter "configuration" alle 8 Unterbereiche angesehen.
Ein Wort "Relais oder Relay" habe ich nicht gefunden, es sei denn, man müsste dazu in der jeweiligen Gruppe weitere Unterpunkte öffnen. Aber das wollte ich lieber nicht machen.

Schon mal danke und Grüße
Werner

**Comment by Jason2866** [comment]
Geh mal bei Configure Module rein.  Steht da Module type `Sonoff Basic`? Wenn ja wechsle hier auf Module type `Generic`. Müsste neu starten und dann sollte der Toggle Button weg sein. Wenn wider erwarten das Teil nicht mehr funktioniert, dann hier wieder zurück ändern auf Sonoff Basic. Denke ich aber ist nicht notwendig.
Der Ersteller der Firmware hat sich keine Mühe gegeben...

**Reply by Fokker-1** [reply]
> Geh mal bei Configure Module rein. Steht da Module type `Sonoff Basic`? Wenn ja wechsle hier auf Module type `Generic`. Müsste neu starten und dann sollte der Toggle Button weg sein. Wenn wider erwarten das Teil nicht mehr funktioniert, dann hier wieder zurück ändern auf Sonoff Basic. Denke ich aber ist nicht notwendig. Der Ersteller der Firmware hat sich keine Mühe gegeben...

Hallo,

super, geanau das war es - herzlichen Dank!

Sieht jetzt aus wie erwünscht (und sinnvoll):

![Tasmota Hichi neue Oberfläche_verdeckt_2024-08-08](https://github.com/user-attachments/assets/3a7d20fd-c1b9-4a83-8050-ce6e959a5f32)

Deine Einschätzung zum Ersteller kann ich nur teilen.

Noch mal danke und Grüße
Werner
